csharp/ql/test/query-tests/Security Features/CWE-117/LogForgingSimpleTypes.cs

```diff
@@ -0,0 +1,46 @@
+using System;
+using System.Diagnostics;
+using System.IO;
+using System.Net;
+using System.Web;
+using Microsoft.Extensions.Logging;
+
+class ILogger
+{
+    public void Warn(string message) { }
+}
+
+enum TestEnum
+{
+    TestEnumValue
+}
+
+public class LogForgingSimpleTypes
+{
+    public void Execute(HttpContext ctx)
```

Comment:
The test cases doesn't test the simple type sanitization as the input to the logger are hardcoded constants and not a data flow source for the log forging query (maybe use the same pattern as in https://github.com/github/codeql/blob/main/csharp/ql/test/query-tests/Security%20Features/CWE-117/LogForging.cs)

Reply:
Interestingly, I have seen hardcoded Enums trigger this, but I will address this to use a true data flow source.